docs/client-metrics.md

```diff
@@ -206,12 +206,17 @@ The event stream is a log of events and the time they occurred while the user is
 * signup.email-optin.visible.false - email opt-in is not visible
 * signup.checkbox.change.marketing-email-optin.checked - user checked the email opt-in
 * signup.checkbox.change.marketing-email-optin.unchecked - user unchecked the email opt-in
-* signup.checkbox.change.customize-sync.checked - user checked the "Choose what to sync" checkbox.
-* signup.checkbox.change.customize-sync.unchecked - user unchecked the "Choose what to sync" checkbox.
+* signup.checkbox.change.customize-sync.checked - user checked the "Choose what to sync" checkbox
+* signup.checkbox.change.customize-sync.unchecked - user unchecked the "Choose what to sync" checkbox
 * error.signup.auth.1012 - user did not enter year of birth
 * error.signup.auth.1016 - user did not enter their birthday
 * error.signup.auth.1029 - signup is disabled on Fx for iOS v1
-* error.signup.auth.1030 - signup has been force disabled by a Sync based relier.
+* error.signup.auth.1030 - signup has been force disabled by a Sync based relier
+* experiment.pw-strength.ALL_NUMBERS_LETTERS - password contains all numbers or letters
+* experiment.pw-strength.BLOOMFILTER_HIT - password was found in bloomfilter
+* experiment.pw-strength.BLOOMFILTER_MISS - password was not found in bloomfilter
+* experiment.pw-strength.PASSWORD_TOO_SHORT - password was less than 8 characters
+* experiment.pw-strength.MISSING_PASSWORD - no password supplied
 
```

Comment:
@TDA - we might as well be consistent with other metrics, can you lowercase the appended status message [before we log it](https://github.com/mozilla/fxa-content-server/pull/2810/files#diff-1c5e553272303abc152fd2aa8fd2b36aR44) and update the tests & docs?

nit: each of these events has a `signup.` prefix. Can you add those, then move the events above teh errors?